lib/ansible/module_utils/network/fortios/fortios.py

```diff
@@ -33,6 +33,7 @@
 
 from ansible.module_utils._text import to_native
```

Comment:
```suggestion
from ansible.module_utils._text import to_text
```

And change instances of `to_native` to `to_text`. They all go through ansible functions, you don't need `to_native` here.